cmd/libs/go2idl/client-gen/generators/fake/generator_fake_for_type.go

```diff
@@ -197,7 +225,7 @@ type Fake$.type|publicPlural$ struct {
 `
 
 var resource = `
-var $.type|allLowercasePlural$Resource = $.GroupVersionResource|raw${Group: "$.groupName$", Version: "$.version$", Resource: "$.type|allLowercasePlural$"}
+var $.type|allLowercasePlural$Resource = $.GroupVersionResource|raw${Group: "$.groupName$", Version: "$.version$", Resource: "$.resource$"}
```

Comment:
Passing the type to a function (namer), as was done previously, is the correct way to use this. Please add a new namer that provides your new logic. Sorry to be a pain about this but last time this got into these generators wrong it took me ages to get it corrected.